features/profile/view/References.test.tsx

```diff
@@ -137,7 +142,7 @@ describe("References", () => {
       });
       userEvent.click(
         screen.getByRole("option", {
-          name: referencesFilterLabels(t)[ReferenceType.REFERENCE_TYPE_FRIEND],
```

Comment:
This comment applies to pretty much everything else I updated in this test suite, but the main reason for updating them is because these tests didn't catch the mistake in the first place, as they were clicking things using code that might be wrong (i.e. the `referencesFilterLabels` and `referenceBadgeLabel`)! This only becomes more obvious when I look into why the tests did nothing 😅